src/components/Circles/PostsList.tsx

```diff
@@ -29,6 +29,7 @@ export const PostsList = ({ circleTile, onOpenPost }: PostsListProps) => {
     useInfinitePosts({
       circleId: circleTile?.circleId,
     });
+  // useFocusEffect(() => {});
```

Comment:
Is this needed?

Reply:
I have a fast follow that removes this almost ready for review.